pages/packages/private_storage.md

```diff
@@ -0,0 +1,42 @@
+# Link private storage
+
+This page provides details on how to link your private Amazon Web Services (AWS) Simple Storage Service (S3) storage to Buildkite Packages within your Buildkite organization. This process can only be conducted by [Buildkite organization administrators](/docs/packages/permissions#manage-teams-and-permissions-organization-level-permissions).
+
+By default, Buildkite Packages provides its own storage to house any packages, container images and modules stored in registries. You can also link your own private AWS S3 bucket to Buildkite Packages, which allows you to:
+
+- Manage Buildkite registry packages, container images and modules stored within your private AWS S3 bucket (that is, your _private storage_). Private storage located closer to your geographical location may provide faster registry access.
+- Use Buildkite Package's management and metadata-handling features to manage these files in registries within your private storage.
+- Maintain control, ownership and sovereignty over the packages, container images and modules stored within your Buildkite Packages registries.
+
+Buildkite Packages uses [AWS CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html) to provision its services within your private AWS S3 storage.
+
+To link your private AWS S3 storage to Buildkite Packages:
```

Comment:
I think we should be really clear in here that you need an existing S3 bucket or to create an S3 bucket before starting the process. The CloudFormation template only adds permissions for Buildkite to interact with the existing bucket, it doesn't create a bucket.

Reply:
Good point! I'll add a **Before you start** section to this page.